src/components/groups/TeamProfile.tsx

```diff
@@ -0,0 +1,51 @@
+import { useRouter } from 'next/router';
+import NextLink from 'next/link';
+import { TabsMenu, TabsMenuItem } from '@taskany/bricks';
+import styled from 'styled-components';
+import { gapL } from '@taskany/colors';
+
+import { useGroup } from '../../api-client/groups/group-api-hook';
+import { CommonHeader } from '../CommonHeader';
+import { FiltersPanel } from '../FiltersPanel';
+
+const StyledTabsMenu = styled(TabsMenu)`
+    margin-left: ${gapL};
+`;
+
+export const TeamProfile = () => {
+    const router = useRouter();
+    const { groupId } = router.query;
+    const userQuery = useGroup(String(groupId));
+    const group = userQuery.data;
+    const parentId = group?.parentId;
+    const parentQuery = useGroup(String(parentId));
+    const parentGroup = parentQuery.data;
+    if (!parentGroup) return null;
+    if (!group) return null;
+
+    const tabsMenuOptions: Array<[string, string]> = [
+        ['People', '/people'],
```

Comment:
that should be a const in a file so it can be imported, not copied

Reply:
done